### ISSUE TITLE
Define the resource memory and cpu requests 

### ISSUE THREAD

**Issue by tppalani** [kedacore/keda#5295]
### Proposal

I want to take this opportunity to define resources limit as part of Keda recommendations while deploying yaml file this will help us to define right size instead of eating whole nodes when the pod is occupied all room space within node

### Scaler Source

Quote define advise policy while installing kubernetes deployment yaml files

### Scaling Mechanics

When there is no quote request defined in deployment yaml Keda should be stop applying the yaml file into the cluster or provide suggestions to set the quote limits

### Authentication Source

NA

**Comment by JorTurFer** [comment]
Hello @tppalani 
I'm not getting your request. Could you rephrase it?

What do you mean with `When there is no quote request defined in deployment yaml Keda should be stop applying the yaml file into the cluster or provide suggestions to set the quote limits`?

KEDA doesn't deploy any deployment into the cluster, KEDA just exposes metrics for scaling them, but the deployment has been already deployed by any other process.

I reckon that I'm missing something

**Comment by tppalani** [comment]
> Hello @tppalani I'm not getting your request. Could you rephrase it?
> 
> What do you mean with `When there is no quote request defined in deployment yaml Keda should be stop applying the yaml file into the cluster or provide suggestions to set the quote limits`?
> 
> KEDA doesn't deploy any deployment into the cluster, KEDA just exposes metrics for scaling them, but the deployment has been already deployed by any other process.
> 
> I reckon that I'm missing something

Sorry for the confusion
As you mentioned Keda exposes metrics for scaling kubernetes objects lets say expand the pod based on certain events,  same way Keda have to analysis’s the namespaces inside the kubernetes cluster and it provide the cpu and memory quotes suggestions to the user like goldilocks

**Comment by JorTurFer** [comment]
I think that it could be and interesting point but out of KEDA scope due to some reasons:
- recommending is not scaling and it requires constant analysis over the resources (KEDA doesn't monitor them but exposes metrics for the HPA controller)
- KEDA doesn't manage the workload definitions and it just generates the HPA and exposes external metrics

In the other hand, maybe the VPA Controller in recommendation mode could help with this if we add the support for it

WDYT @kedacore/keda-core-contributors ?
